README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About me
+
+My name is Yanwen. I am currently a **MAPSS** student in Psychology. 
+Before this course, I have had some experience with:
+* R
+  * ggplot2
+* SPSS
+* mplus
+But I hope to learn programming systematically in this course.
+
+I am not familiar with Github yet but I will be telling you more about me. Here is my page https://github.com/yw34.
```

Comment:
You could include a hyperlink to your page by using the following syntax

[linked phrase](http://example.com)
[my page](https://github.com/yw34)